src/ui/components/Library/Team/View/NewTestRuns/TestRunsPage.tsx

```diff
@@ -144,9 +145,13 @@ function TestRunsContent() {
               data-filtered-by-text={filterByText}
               data-test-id="TestRunList"
             >
-              <Suspense fallback={<LibrarySpinner />}>
+              {testRunsLoading ? (
+                <div className="flex h-full items-center justify-center">
+                  <LibrarySpinner />
+                </div>
+              ) : (
                 <TestRunList />
-              </Suspense>
```

Comment:
Remove suspense because underlying data (testRuns) is loaded with useImperativeCacheValue in context root, that means no suspense. So I think no point of having that.